hackathon_site/registration/models.py

```diff
@@ -28,13 +28,13 @@ def __str__(self):
 
 
 class Application(models.Model):
-    GENDER_CHOICES = [
+    PRONOUN_CHOICES = [
         (None, ""),
-        ("male", "Male"),
-        ("female", "Female"),
-        ("non-binary", "Non-binary"),
-        ("other", "Other"),
-        ("no-answer", "Prefer not to answer"),
+        ("he/him", "he/him"),
```

Comment:
I'm not sure how well django handles slashes for database columns, could you change the / to a - in the first element of the tuple e.g. he-him